sites/all/modules/reol_frontend/reol_frontend.module

```diff
@@ -281,6 +281,49 @@ function reol_frontend_form_profile2_edit_provider_publizon_form_alter(&$form, &
 
   // Hide sms fee.
   $form['ding_user_fee_sms']['#access'] = FALSE;
+
+  // Add message about confirming values right under “Support id”.
+  $message = variable_get('ereol_profile_confirm_message');
+  if (!empty($message['value'])) {
+    $form['ereol_profile_confirm_message'] = array(
+      '#markup' => '<div class="profile-confirm-message">' . $message['value'] . '</div>',
+      '#weight' => -49,
```

Comment:
Optional: As you are manipulating the form, you could add "state" and make them hidden when no value is set.